### ISSUE TITLE
[BUG] Job progress bar does not reach 100% for idle processes

### ISSUE THREAD

**Issue by MBartkowiakSTFC** [ISISNeutronMuon/MDANSE#584]
**Description of the error**
Normally, a job should reach 100% when it is finished. However, if more processes than necessary are created in multiprocessing mode, it can happen that 100% is not reached.

**Describe the expected result**
For a trajectory with a single molecule, Infrared analysis in multiprocessing mode should reach 100% when finished.

**Describe the actual result**
The job table was showing 0% and 'Finished' at the end of the run.

**Suggested fix**
One sensible solution would be not to create more processes than there are analysis steps to be run.

**Additional details**
N/A

**Comment by oerc0122** [comment]
@ChiCheng45 is this fixed by #768

**Comment by ChiCheng45** [comment]
> [@ChiCheng45](https://github.com/ChiCheng45) is this fixed by [#768](https://github.com/ISISNeutronMuon/MDANSE/pull/768)

Nope, but I do think it was fixed by #779. @MBartkowiakSTFC I think we can close this after #779 gets merged.

**Comment by MBartkowiakSTFC** [comment]
Testing shows that this is fixed now.
